fix compilation error:

/usr/include/tclUnixPort.h:112:33: error:
../compat/unistd.h: No such file or directory

--- expect-5.42/Dbg.c.orig	2002-03-23 05:11:54.000000000 +0100
+++ expect-5.42/Dbg.c	2005-06-06 22:27:01.000000000 +0200
@@ -22,6 +22,7 @@
 #endif
 
 
+#include "expect_cf.h"
 #include "tclInt.h"
 /*#include <varargs.h>		tclInt.h drags in varargs.h.  Since Pyramid */
 /*				objects to including varargs.h twice, just */
